### ISSUE TITLE
Unable to get it working in a Vue project. 

### ISSUE THREAD

**Issue by Marchiuzzz** [jimp-dev/jimp#1283]
# Expected Behavior
Following the docs, I should be able to use Jimp out of the box

# Current Behavior
I get a `TypeError: Jimp is not a constructor` or `TypeError: jimp__WEBPACK_IMPORTED_MODULE_1___default(...) is not a constructor` error.

## Context
When importing Jimp like this:
`import Jimp from "jimp"`
I get `TypeError: jimp__WEBPACK_IMPORTED_MODULE_1___default(...) is not a constructor` error

I have also tried using it like so `var Jimp = require("jimp")` but then I get `TypeError: Jimp is not a constructor`

This is the full code

```
import _ from "lodash"
import Jimp from "jimp"

class ImageMaker {
	static make_money_img(coins, height) {
		// var Jimp = require("jimp")  <---- tried this as well 
		let coinwidth = 100
		let coinheight = coinwidth * 0.55
		let coincount = coins.length
		let matrix_width = Math.ceil(coincount / height)
		let x = matrix_width
		let y = height

		let img_w = coinwidth * x
		let img_h = coinheight * y

		var mainImage = new Jimp(img_w, img_h, 0x0, (err, image) => {
			// ... my code here
		})
		return mainImage.getBase64(Jimp.MIME_PNG)
	}
}

export { ImageMaker }

```

- Jimp Version:  ^0.22.12
- Operating System: Windows 11
- Node version: v14.21.3

**Comment by supersuryaansh** [comment]
Same issue, any luck?

**Comment by Marchiuzzz** [comment]
@supersuryaansh Yeah, I managed to get it working using
```
import "jimp"
const { Jimp } = window
```

**Comment by supersuryaansh** [comment]
@Marchiuzzz thanks, I got it working with this:
import * as _Jimp from 'jimp';
const Jimp = (typeof self !== 'undefined') ? (self.Jimp || _Jimp) : _Jimp;

**Comment by meetfilipe** [comment]
Very interesting, @Marchiuzzz solution seems cleaner but you need to cheat a little bit adding a `ts-ignore`.
@supersuryaansh solution seems more cryptic but it's actually the best solution.

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in [`v1.0.2`](https://github.com/jimp-dev/jimp/releases/tag/v1.0.2) :rocket:
